dags/fxci_metric_export.py

```diff
@@ -0,0 +1,74 @@
+"""
+Exports Firefox-CI worker data from the Google Cloud Monitoring to BigQuery.
+
+The container is defined in [fxci-etl](https://github.com/mozilla-releng/fxci-etl).
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.providers.cncf.kubernetes.secret import Secret
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+default_args = {
+    "owner": "ahalberstadt@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2024, 7, 8),
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+project_id = "moz-fx-data-shared-prod"
+
+tags = [Tag.ImpactTier.tier_3]
+
+env_vars = {
+    "FXCI_ETL_BIGQUERY_PROJECT": project_id,
+    "FXCI_ETL_BIGQUERY_DATASET": "fxci",
+    "FXCI_ETL_STORAGE_PROJECT": "moz-fx-dev-releng",
+    "FXCI_ETL_STORAGE_BUCKET": "fxci-etl",
+}
+
+secrets = [
+    Secret(
+        deploy_type="env",
+        deploy_target="FXCI_ETL_STORAGE_CREDENTIALS",
+        secret="airflow-gke-secrets",
+        key="fxci-etl-gcp-credentials",
```

Comment:
For consistency with other DAGs:
```suggestion
        key="fxci_etl_secret__gcp-credentials",
```